klayout/drc/rule_decks/main.drc

```diff
@@ -14,880 +14,763 @@
 # limitations under the License.
 ################################################################################################
 
-#===========================================================================================================================
-#------------------------------------------- GF 0.18um MCU DRC RULE DECK --------------------------------------------------
-#===========================================================================================================================
+#=======================================================================================================================
+#--------------------------------------------- GF 0.18um MCU DRC RULE DECK ---------------------------------------------
+#=======================================================================================================================
 require 'time'
-require "logger"
+require 'logger'
 require 'etc'
 
 exec_start_time = Time.now
 
-logger = Logger.new(STDOUT)
+logger = Logger.new($stdout)
 
-logger.formatter = proc do |severity, datetime, progname, msg|
-  "#{datetime}: Memory Usage (" + `pmap #{Process.pid} | tail -1`[10,40].strip + ") : #{msg}
+logger.formatter = proc do |_severity, datetime, _progname, msg|
+  "#{datetime}: Memory Usage (" + `pmap #{Process.pid} | tail -1`[10, 40].strip + ") : #{msg}
 "
 end
 
 #================================================
 #----------------- FILE SETUP -------------------
 #================================================
-logger.info("Starting running GF180MCU Klayout DRC runset on %s" % [$input])
-logger.info("Ruby Version for klayout: %s" % [RUBY_VERSION])
+logger.info("Starting running GF180MCU Klayout DRC runset on #{$input}")
+logger.info("Ruby Version for klayout: #{RUBY_VERSION}")
 
 if $input
-    if $topcell
-        source($input, $topcell)
-    else
-        source($input)
-    end
-end
-
-if $table_name
```

Comment:
any reaosn we got rid of the table_name check here?

Reply:
Added directly to `tail.drc` file where it is used

Reply:
I don't see it in `tail.drc`, did you push?

Reply:
Yes
![image](https://user-images.githubusercontent.com/91015308/217475446-6a168f1e-717e-4790-b3e4-5a99a83c33af.png)